soak_bench_scripts/benchmark.sh

```diff
@@ -1,31 +1,36 @@
 #!/bin/bash
-#set -e
+# set -e
 
 date
 
 # This code is obsolete but leaving just in case there is a need for graphs again
 # Remove old graphs
-#find /tmp/parsec_graphs -maxdepth 1 -type d -ctime 1 -exec rm -r "{}" \;
+# Find /tmp/parsec_graphs -maxdepth 1 -type d -ctime 1 -exec rm -r "{}" \;
```

Comment:
Same for this, although this is slightly more justifiable to leave hanging around since it's a more complex command.  Your call.

Reply:
Hmm, I was erring on the side of caution with this one - since I've been running the scripts had no occasion to use this but was leaving it in thinking "just in case I do need it again".